docs/Microservices/RPC/buildRPC_Manual.md

```diff
@@ -0,0 +1,228 @@
+    cmake --build .
+    ```
+
+=== "CentOS7"
+    !!! important
+        **The "export PATH=..." (for CMake and Git), and "scl enable devtoolset-7 bash" (for GCC) must be executed in in the Terminal you are building from; i.e. these are temporary changes which only affect the current Terminal session.**
+
+    **Install Dependencies**
+
+    1. Download CMake.
+    ``` bash
+    ./cmake-3.10.2-Linux-x86_64.sh
+    export PATH=/home/user/Downloads/cmake-3.10.2-Linux-x86_64/bin:$PATH
+    ```
+
+    2. Update GCC toolchain.
+    ```
+    sudo yum install centos-release-scl
+    sudo yum install devtoolset-7
+    scl enable devtoolset-7 bash
+    ```
+
+    3. Build Git source control system.
+    ``` bash
+    sudo yum install curl-devel expat-devel gettext-devel openssl-devel zlib-develperl-CPAN perl-devel
+    git clone https://github.com/git/git.git
+    make configure
+    make
+    export PATH=/home/user/Downloads/git:$PATH
+    ```
+
+    **Build VCPKG and C++ REST SDK**
+        
+    1. In the `rpc` directory, clone the VCPKG repository. Vcpkg is a C/C++ Library Manager for Windows that was created by Microsoft.  Find out more about it [here](https://github.com/microsoft/vcpkg).
+    ``` bash
+    git clone --branch 2020.11-1 https://github.com/microsoft/vcpkg.git && cd vcpkg
+    ```
+
+    3. Build vcpkg.exe using the following command.
+    ``` bash
+    ./bootstrap-vcpkg.sh
+    ```
+        
+    4. Install C++ REST SDK. This can take anywhere from 8 - 15 minutes depending on download speeds and installation times.
+    ``` bash
+    ./vcpkg install cpprestsdk[websockets]
+    ```
+
+    **Build RPC**
+
+    1. Return to the `rpc` directory and create a new 'build' directory.
+    ``` bash
+    cd .. && mkdir build && cd build
+    ```
+    
+    2. Generate the CMake config
+    ``` bash
+    cmake -DCMAKE_TOOLCHAIN_FILE=../vcpkg/scripts/buildsystems/vcpkg.cmake -DCMAKE_BUILD_TYPE=Release -DNO_SELECT=ON ..
+    ```
+
+        !!! note
+            RPC can also be built in a non-production debug mode rather than release using the following command. The debug mode includes debug symbols.
+            ```
+            cmake -DCMAKE_TOOLCHAIN_FILE=../vcpkg/scripts/buildsystems/vcpkg.cmake -DCMAKE_BUILD_TYPE=Debug -DNO_SELECT=ON ..
+            ```
+
+    3. Build the RPC executable
+    ```bash
+    cmake --build .
+    ```
+
+### Run RPC
+
+For additional information on possible arguments when invoking RPC, see [Command Examples](commandsRPC.md).
+
+The following example command shows how to activate and configure an Intel AMT device using a pre-defined profile on your local network.
+
+=== "Windows"
+    ```
+    rpc --url wss://localhost:8080 --cmd "-t activate --profile profile1"
+    ```
+=== "Linux"
+    ``` bash
+    sudo ./rpc --url wss://localhost:8080 --cmd "-t activate --profile profile1"
+    ```
+
+
+Example Success Output:
+
+[![RPC Success](../../assets/images/RPC_Success.png)](../../assets/images/RPC_Success.png)
```

Comment:
An updated image needs to be inserted here as RPC no longer outputs verbose messages such as this with the above command line arguments. It currently only outputs dots (...) as the activation progresses and in the case were the --nocertcheck flag is provided a message indicating the skipping of the cert check is output.

Reply:
This will be handled in [this PR](https://github.com/open-amt-cloud-toolkit/docs/pull/15) with the rest of the updated 1.1 screenshots.